src/logic/collectibles/utils/index.ts

```diff
@@ -0,0 +1,122 @@
+import { getERC721TokenContract, getStandardTokenContract } from 'src/logic/tokens/store/actions/fetchTokens'
+import { TokenState } from 'src/logic/tokens/store/reducer/tokens'
+import { isTokenTransfer } from 'src/logic/tokens/utils/tokenHelpers'
+import { sameAddress } from 'src/logic/wallets/ethAddresses'
+import { CollectibleTx } from 'src/routes/safe/components/Balances/SendModal/screens/ReviewCollectible'
+
+// CryptoKitties Contract Addresses by network
+// This is an exception made for a popular NFT that's not ERC721 standard-compatible,
+//  so we can allow the user to transfer the assets by using `transferFrom` instead of
+//  the standard `safeTransferFrom` method.
+export const CK_ADDRESS = {
+  [ETHEREUM_NETWORK.MAINNET]: '0x06012c8cf97bead5deae237070f9587f8e7a266d',
+  [ETHEREUM_NETWORK.RINKEBY]: '0x16baf0de678e52367adc69fd067e5edd1d33e3bf',
+}
+
+// safeTransferFrom(address,address,uint256)
+export const SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH = '42842e0e'
+
+/**
+ * Verifies that a tx received by the transaction service is an ERC721 token-related transaction
+ * @param {TxServiceModel} tx
+ * @param {string | undefined} txCode
+ * @param {TokenState | undefined} knownTokens
+ * @returns boolean
+ */
+export const isSendERC721Transaction = (tx: TxServiceModel, txCode?: string, knownTokens?: TokenState): boolean => {
+  // "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85" - ens token contract, includes safeTransferFrom
+  // but no proper ERC721 standard implemented
+  return (
+    (txCode?.includes(SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH) &&
+      tx.to !== '0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85') ||
+    (isTokenTransfer(tx) && !knownTokens?.get(tx.to))
+  )
+}
+
+/**
+ * Returns the symbol of the provided ERC721 contract
+ * @param {string} contractAddress
+ * @returns Promise<string>
+ */
+export const getERC721Symbol = async (contractAddress: string): Promise<string> => {
+  let tokenSymbol = 'UNKNOWN'
+
+  try {
+    const ERC721token = await getERC721TokenContract()
+    const tokenInstance = await ERC721token.at(contractAddress)
+    tokenSymbol = tokenInstance.symbol()
+  } catch (err) {
+    console.error(`Failed to retrieve token symbol for ERC721 token ${contractAddress}`)
+  }
+
+  return tokenSymbol
+}
+
+/**
+ * Verifies if the provided contract is a valid ERC721
+ * @param {string} contractAddress
+ * @returns boolean
+ */
+export const isERC721Contract = async (contractAddress: string): Promise<boolean> => {
+  const ERC721Token = await getStandardTokenContract()
+  let isERC721 = false
+
+  try {
+    await ERC721Token.at(contractAddress)
+    isERC721 = true
+  } catch (error) {
+    console.warn('Asset not found')
+  }
+
+  return isERC721
+}
+
+/**
+ * Returns a method identifier based on the asset specified and the current network
+ * @param {string} contractAddress
+ * @returns string
+ */
+export const getTransferMethodByContractAddress = (contractAddress: string): string => {
```

Comment:
Could you add two tests cases? one for `contractAddress = Criptokitties` and other for a random collectible address

Reply:
done, thanks for the help! [`6e96f9d` (#1574)](https://github.com/gnosis/safe-react/pull/1574/commits/6e96f9d9191bfde5f11097a0054b0fe071d53385)